sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 ## Release History
-
-### 1.3.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
-#### Other Changes
+### 1.3.0 (2022-08-15)
```

Comment:
Just noticed we have the wrong header size on the line above and for every version. We should use `# Release History` above and the following below:
```suggestion
## 1.3.0 (2022-08-15)
```